src/libtls/tls_peer.c

```diff
@@ -1195,14 +1201,14 @@ static status_t send_client_hello(private_tls_peer_t *this,
 								  bio_writer_t *writer)
 {
 	tls_cipher_suite_t *suites;
-	bio_writer_t *extensions, *curves = NULL, *versions, *key_share, *signatures;
+	bio_writer_t *extensions, *curves = NULL, *versions, *signatures;
+	bio_writer_t *ext_content;
```

Comment:
I guess you could also just keep the existing name for this writer, or use `ext_content` also for the other extensions.